services/order.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User, MovieSession
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> Order:
+    user = User.objects.get(username=username)
+    order = Order.objects.create(user=user)
+    if date:
+        order.created_at = datetime.strptime(
+            date, "%Y-%m-%d %H:%M"
+        )
+        order.save()
+    for ticket in tickets:
+        Ticket.objects.create(
+            row=ticket["row"],
+            seat=ticket["seat"],
+            movie_session=MovieSession.objects.get(
+                id=ticket["movie_session"]
+            ),
+            order=order,
+        )
+    return order
+
+
+def get_orders(username: str = None) -> QuerySet:
```

Comment:
You could indicate that you return a `QuerySet` of `Order` objects in the return type annotation. Please, make changes to the return type annotation in all relevant places to encounter that.